### DISCUSSION TITLE
Full-link trace

### DISCUSSION THREAD

**Discussion by wanghongda-0624** [getsentry/support#80] [Support]
### Problem Statement

I found a trace-id in sentry, i want to know does backend can capture this trace-id,  if not, is there any method or configurations can be used to get the trace-id, thanks a lot.

**Comment by souredoutlook** [comment]
Hey @wanghongda-0624 - can you tell me more about your use case? Each transaction will have an associated trace-id. It's possible to share the same trace-id across to transactions if they are part of the same trace.

I'd like to know more about what you want to accomplish so I can point you to the correct resources.

Let me know :)
